### PR TITLE
Create README.fr.md file

### PR DESCRIPTION
**Pull request by pshipton**
Proposal of a french translation for the readme.md file. This translation can be accessed by a link in the original file to a new file named readme.fr.md.

### CODE REVIEW

**Comment by pshipton** [comment]
@Yas01 before we can accept any change from you, you need to sign the ECA and sign-off your commit as outlined in https://github.com/eclipse/openj9/blob/master/CONTRIBUTING.md

File: README.fr.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
+<!--
+Copyright (c) 2017, 2019 IBM Corp. and others
+
+Ce programme et le matériel d'accompagnement sont disponibles sous
```
Comment (pshipton) [inline README.fr.md]: The license shouldn't be translated, it should remain in English.

**Comment by Yas01** [comment]
@pshipton, I still have a problem with the "eclipsefdn/eca " check. However I've already sign the ECA and sign-off my commit.

**Comment by pshipton** [comment]
@Yas01 there is no `Signed-off-by:` line on either commit attached to this PR.

**Comment by Yas01** [comment]
image
@pshipton I've already signed it! but still have this problem !

**Comment by pshipton** [comment]
@Yas01 none of the commits have the `Signed-off-by:` line. Once you figure out how to fix it, you'll have to squash your commits to remove the commits which are not signed.
image

**Comment by pshipton** [comment]
This seems to have been dropped so I'll close it for now.

**Pull request closed without merging**